src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -502,8 +502,8 @@ public void handle(Endpoint<?> endpoint, Handler<?> handler) {
         JsonObject bodyAsJson = EMPTY_JSON;
         try {
             // check for an empty body
-            if (routingContext.getBody() != null && routingContext.getBody().length() != 0) {
-                bodyAsJson = routingContext.getBodyAsJson();
+            if (routingContext.body().buffer() != null && routingContext.body().buffer().length() != 0) {
+                bodyAsJson = routingContext.body().asJsonObject();
```

Comment:
Everywhere else you are using `.body().buffer().....` why do you not need that here? Dos the json method handle that for you?

Reply:
The alternative suggested for routingContext.getBodyasJson() was routingContext.body().asJsonObject().

Reply:
As I can see from the current implementation here https://github.com/vert-x3/vertx-web/blob/master/vertx-web/src/main/java/io/vertx/ext/web/RequestBody.java#L131 you could change the if condition just checking `rounting.body().isEmpty()` instead of checking null and length on it.